service-api/src/main/java/greencity/dto/habit/HabitAssignManagementDto.java

```diff
@@ -0,0 +1,34 @@
+package greencity.dto.habit;
+
+import java.time.ZonedDateTime;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import lombok.*;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@EqualsAndHashCode
+@Builder
+public class HabitAssignManagementDto {
+    @NotNull
+    @Min(1)
+    private Long id;
+    @NotEmpty
```

Comment:
NotNull annotaion probably will be enough for not String fields

Reply:
Changes done.